package.json

```diff
@@ -7,7 +7,7 @@
     "audit_deps": "node ./build/commands/scripts/audit.js -- --audit_dev_deps",
     "check_security": "node ./build/commands/scripts/checkSecurity.js",
     "cibuild": "node ./build/commands/scripts/commands.js cibuild",
-    "init": "npm --prefix ../../ run init --",
+    "init": "cd ../../ && npm run init --",
```

Comment:
This is not cross-platform, is it?

Reply:
@atuchin-m says this does work on Windows, so gtg.

Reply:
Unless it only works in one shell and not the other (cmd.exe vs. powershell)?

Reply:
Checked this for `cmd.exe` and `powershell`, all is fine.
To run command nodejs spawn an additional `cmd.exe` process which execute our commands.
Example for ` "create_dist": "**cmd** && node ./build/commands/scripts/commands.js create_dist",`
![image](https://user-images.githubusercontent.com/45488748/143399182-6f04c9da-f15a-44e2-8f91-67c73f85fdff.png)